VoxelSniperCore/src/main/java/com/github/kevindagame/brush/BlockResetSurfaceBrush.java

```diff
@@ -100,13 +99,13 @@ private void applyBrush(final SnipeData v) {
         }
     }
 
-    private boolean checkBlock(IWorld world, int x, int y, int z, boolean airFound) {
+    private boolean checkBlock(IWorld world, int x, int y, int z) {
         IBlock block = world.getBlock(this.getTargetBlock().getX() + x + 1, this.getTargetBlock().getY() + y, this.getTargetBlock().getZ() + z);
         if (block.getMaterial().isAir()) {
             resetBlock(block.getLocation());
```

Comment:
Actually looking at this, why are we resetting the block here, @KevinDaGame Do you know?

Reply:
I have no clue, but it was in there before

Reply:
Reset block resets the data, which air shouldn't have (because it is.. air), I do not think this would have much effect if any.

Reply:
I think we could safely remove it

Reply:
This behaviour is in the master branch too: https://github.com/KevinDaGame/VoxelSniper-Reimagined/blob/master/VoxelSniperCore/src/main/java/com/github/kevindagame/brush/BlockResetSurfaceBrush.java#L86

Reply:
Lets keep it for now, I might make a separate PR to remove that line.

Reply:
THis can't be removed, since this will update lightlevels